MockKitUI/MockPumpManager+UI.swift

```diff
@@ -31,7 +31,7 @@ extension MockPumpManager: PumpManagerUI {
         return MockHUDProvider(pumpManager: self)
     }
 
-    public static func createHUDViews(rawValue: [String : Any]) -> [BaseHUDView] {
+    public static func createHUDViews(rawValue: [String : Any]) -> [LevelHUDView] {
```

Comment:
If the Loop end usage of this protocol only uses one view, then the method signature should communicate that.